app/views/blog/show.html.erb

```diff
@@ -1,5 +1,5 @@
-<h1><%  @blog.title %></h1>
+<h1><%= @blog.title %></h1>
 
 </br>
-<h1><%  @blog.content %></h1>
-<p><%= link_to 'All the Blogs', blogs_path %></p>
\ No newline at end of file
+<h1><%=  @blog.content %></h1>
+<h1><p><%= link_to 'All the Blogs', blogs_path %></p></h1>
```

Comment:
It looks like you have a paragraph tag nested in an h1, you may want to choose one or the other in this case.